packages/state-manager/src/sagas/identity/identity.transform.ts

```diff
@@ -0,0 +1,16 @@
+import { createTransform } from 'redux-persist'
+import { StoreKeys } from '../store.keys'
+import { IdentityState } from './identity.slice'
+
+export const IdentityTransform = createTransform(
+  (inboundState: IdentityState, _key) => {
+    return { ...inboundState }
+  },
+  (outboundState: IdentityState, _key) => {
+    return {
+      ...outboundState,
+      isRegisterButtonClicked: false
```

Comment:
I really don't like it that we keep in store the information about component's state